README.md

```diff
@@ -19,7 +19,7 @@ started, leave a comment on the issue or ask around [on IRC](https://wiki.whatwg
 ### Pull requests
 
 In short, change `dom.bs` and submit your patch, with a
-[good commit message](https://github.com/erlang/otp/wiki/Writing-good-commit-messages). Consider
+[good commit message](https://github.com/whatwg/meta/blob/master/COMMITTING.md). Consider
 reading through the [WHATWG FAQ](https://wiki.whatwg.org/wiki/FAQ) if you are new here.
```

Comment:
Update this to https://whatwg.org/faq while you're here?

Reply:
[The JAMS Foundation - Promoting and Innovating Dispute Resolution Worldwide.pdf](https://github.com/whatwg/dom/files/9837666/The.JAMS.Foundation.-.Promoting.and.Innovating.Dispute.Resolution.Worldwide.pdf)